hdf5.spec

```diff
@@ -486,7 +486,7 @@ done
 %{_libdir}/*.a
 
 %files -n java-hdf5
-%{_jnidir}/hdf5.jar
+%{_libdir}/hdf5.jar
```

Comment:
Why is this changing from `%{_jnidir}` to `%{_libdir}`?  They have different values on both EL7 and Leap 15:

```
el7$ rpm --eval %{_jnidir}
/usr/lib/java
el7$ rpm --eval %{_libdir}
/usr/lib64
```

```
leap15$ rpm --eval %{_jnidir}
/usr/lib64/java
leap15$ rpm --eval %{_libdir}
/usr/lib64
```

Reply:
see above

Reply:
Yeah, I think this is an install problem, not that the `%files` needs changing.  The distros want java stuff installed in their %{_jnidir}` so the install should be adjusted to meet that.  The ideal mechanism for that would be a `configure` option to specify where java stuff goes, but short of that, we need to edit the`Makefile` to achieve that.

https://github.com/daos-stack/hdf5/blob/fffb9b729abd7d91969a9810a18594e18a6388f0/hdf5.spec#L234-L239 gives us a hint as to where that is done to see if (a) that location is influenced by a `configure` option and if not, where we can apply an edit to set the correct destination directory.

Reply:
I missed one of the edits when updating the hdf5-build.patch.  This should fix this issue...  Files should install in {_jnidir}